exist-core/src/test/xquery/fn.xql

```diff
@@ -126,3 +126,46 @@ function fnt:tokenize-onearg($str as xs:string) {
      tokenize($str)
 };
 
+declare
+    %test:assertError("err:XPDY0002")
+function fnt:document-uri0_noarg() {
+     document-uri()
+};
+
+declare
+    %test:assertError("err:XPDY0002")
+function fnt:document-uri0() {
+     document-uri(.)
+};
+
+declare
+    %test:assertEquals("/db/fn-test/test.xml")
+function fnt:document-uri0_context() {
+     root(collection('/db/fn-test')//book)/document-uri()
+};
+
+declare
+    %test:assertError("err:XPDY0002")
+function fnt:document-uri0_context_empty() {
+     root(collection('/db/fn-test')//bookies)/document-uri()
+};
+
+declare
+    %test:assertEquals("/db/fn-test/test.xml")
+function fnt:document-uri1() {
+     document-uri(root(collection('/db/fn-test')//book))
+};
+
+declare
+    %test:assertEmpty
+function fnt:document-uri1_empty() {
+     document-uri(root(collection('/db/fn-test')//bookies))
+};
+
+declare
```

Comment:
I think maybe you were searching for `%test:assertTrue` ;-)

```xquery
declare
    %test:assertTrue
function fnt:fn-document-uri-32() {
     let $uri := "/db/fn-test/test.xml"
     return
         fn:contains(fn:doc($uri)/document-uri(), $uri)
};
```

Reply:
:-) made the change blindly :-)

Reply:
Hmmm yes, but I followed the original test, no further modification